ui/src/app/app-routing.module.ts

```diff
@@ -128,8 +128,27 @@ const routes: Routes = [
                   intermediate: true
                 }
               }
-            },
-            {
+            }, {
+              path: 'notebooks/create',
```

Comment:
We're probably going to want to change the behavior here. The create should actually create the notebook, then navigate to the "/notebooks/name" page. Alternatively we change "?create" to be a query parameter and put the name in the URL normally.

The issue with the current approach is that the top bar is going to be "Creating a new Notebook" instead of the notebook name while you're looking at your newly created notebook.

Reply:
When you say "top bar" do you mean in the browser's navigation bar?  Because the "top bar" of our app, shows the breadcrumbs regardless of whether it's a create or not, nothing referring to creating. The redirect page WILL say "creating new notebook" (not in top bar, but at top of page, under top bar) but, that makes sense to me. Can you clarify before I make this change?

Reply:
Huh, you're right that breadcrumbs aren't being shown. I think that may actually just be a bug. Don't need to worry about it for this PR then. Filed RW-1564

Reply:
Hey CH. Closed that bug. Since the notebookRedirectComponent now includes the display of the notebook on redirect, I already addressed breadcrumbs being shown. I think you misunderstood my original comment. The notebook redirect page WILL now show breadcrumbs (see img I attached to bug before closing).